### DISCUSSION TITLE
Importing follows?

### DISCUSSION THREAD

**Discussion by poperigby** [jonjomckay/fritter#102] [Q&A] [answered]
I have quite a few Twitter follows, and it will be quite the pain to follow them one-by-one. Can I somehow export from my Twitter account and import to Fritter? I see there's an import option, but I don't know how to export from Twitter.

**Comment by jonjomckay** [comment, marked as answer]
There's no way in the app currently, but I've opened an issue for it: #143

**Reply by poperigby** [reply]
Thanks!
